src/components/ImportExport.js

```diff
@@ -178,14 +178,18 @@ class ImportExport extends React.Component {
     const trailFeatures = []
     const hydrantFeatures = []
 
-    trails.valueSeq().forEach((v) => {
+    trails
+      .sortBy((a) => a.get('name'))
+      .valueSeq()
+      .forEach((v, trailIndex) => {
+
       let trailName = v.get('name').split(' ').join('_')
       // Iterate through Trail's Features
-      v.get('features').forEach((f) => {
+      v.get('features').forEach((f, fIndex) => {
         if (f.get('originalTrailName')) {
           trailName = f.get('originalTrailName')
         }
-        const description = trailName
+        const description = `${trailName},${trailIndex + fIndex + 1}`
```

Comment:
I guess check with him tomorrow (or just look at the current map files to see if multi-feature trails get the same or not), but I think this should just be `trailIndex + 1`, all the features get the same description.